metatron-app/metatron-discovery
# #2604 Move when clicking on Dashboard and Workbench in Top User

open new window containing dashboard, workbench when click link in top user

### Description

open new window showing workbench or dashboard when click link in top user
**Related Issue** : 
#2604 



### How Has This Been Tested?
1. Exploration -> data
2. click any card
3. click link in top user 
4. check new window is showing properly


#### Need additional checks?


### Types of changes



### Checklist:


### Additional Context
